pages/who-we-are/corporate/annual-report-15-16/04-financial-statement.md

```diff
@@ -0,0 +1,2687 @@
+
+*Intangibles*
+
+The entity’s intangibles comprise of internally developed software for internal use. These assets are carried at cost less accumulated amortisation and accumulated impairment losses.
+
+Amortisation rates apply to intangibles and are based of the useful life of 2 to 3 years.
+
+*Revaluations*
+
+Fair values for each class of asset are determined as shown below:
+
+<table class="content-table" summary="Revaluations">
+  <colgroup>
+  <col />
+  <col />
+  </colgroup>
+  <thead>
+    <tr>
+      <th><em>Asset class</em></th>
+      <th><em>Fair value measurement</em></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Leasehold improvements</td>
+      <td>Depreciated replacement cost</td>
+    </tr>
+    <tr>
+      <td>Plant and equipment</td>
+      <td>Market selling price or depreciated replacement cost</td>
+    </tr>
+  </tbody>
+</table>
+
+*Depreciation*
+
+Depreciable property, plant and equipment assets are written-off to their estimated residual values over their estimated useful lives to the DTO using, in all cases, the straight-line method of depreciation.
+
+Depreciation rates (useful lives), residual values and methods are reviewed at each reporting date and necessary adjustments are recognised in the current, or current and future reporting periods, as appropriate.
+
+Depreciation rates applying to each class of depreciable asset are based on the following useful lives:
+
+<table class="content-table" summary="Depreciation rates applying to each class of depreciable asset are based on the following useful lives:">
+  <colgroup>
+  <col />
+  <col />
+  </colgroup>
+  <thead>
+    <tr>
+      <th />
+      <th><strong>2016</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Leasehold improvements</td>
+      <td>Lease term</td>
+    </tr>
+    <tr>
+      <td>Plant and equipment</td>
+      <td>2 to 10 years</td>
+    </tr>
+  </tbody>
+</table>
+
+*Impairment*
+
+All assets were assessed for impairment during 2016.
+
+Where indications of impairment exist, the asset’s recoverable amount is estimated and an impairment adjustment made if the asset’s recoverable amount is less than its carrying amount.
+
+A write-down of $957,000 was recognised in 2016 for Leasehold Improvements for the write-off of the South Tower of level 3 50 Marcus Clarke Street due to the termination of the lease.
+
+*Derecognition*
+
+An item of property, plant and equipment is derecognised upon disposal or when no further future economic benefits are expected from its use or disposal.
+  </div>
+ </div>
+{% include hx.html lvl=4 text=" " %}  
```

Comment:
Tricky :) 🤦‍♂️ 
As discussed, move "Note 2.2B: Prepayments" out of the table into this heading.